airflow/dags/utils/operators/JupyterOperator.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, file_path, **kwargs):
         self.ui_color = job_colors["ipynb"]
 
         # The volume is shared at the same location with the pythonserver container
-        self.command = '(jupyter nbconvert --to script --execute --stdout %s | python)' % (self.ipynb_file, )
+        self.command = '(jupytext --to py --output - %s | python)' % (self.ipynb_file, )
```

Comment:
Could you support both `jupyter nbconvert` and `jupytext`? It would be nice for folks to have the option.

Additionally - if the `jupytext` line emits an output, can you clean it up? See the `RmdOperator`'s command.

Reply:
> Could you support both jupyter nbconvert and jupytext? It would be nice for folks to have the option.

How do you define support? What usecase?

Jupytext uses nbconvert under the hood, so I think this might be doing what you're asking!

>  if the jupytext line emits an output, can you clean it up?

It doesn't emit output